modules/ROOT/pages/maintenance/user-ids/user-ids.adoc

```diff
@@ -0,0 +1,74 @@
+= Listing User IDs
+:toc: right
+:description: There are rare situations like when using shell commands that require the user ID as parameter. This page shows how to get it.
+
+== Introduction
+
+{description} As an example of a use case, see the xref:{s-path}/search.adoc#manually-trigger-re-indexing-spaces[Manually Trigger Re-Indexing Spaces].
+
+. Via the web UI +
+Currently, there is no option to get a user ID via the web UI.
+
+. Via CLI +
+Administrators can query Infinite Scale via the graph API and get a users ID via shell commands using an authenticated curl GET command accessing the Infinite Scale https://owncloud.dev/apis/[API].
++
+--
+The following prerequisites apply:
+
+* The administrator must have shell access where Infinite Scale runs.
+* The administrator needs to use the active bearer token in the request described below.
+* For ease of reading the result, the https://jqlang.github.io/jq/[jq] library should be installed on the OS where the shell command is executed. It is used in the examples.
+
+Note that the life span of the bearer token is short, in  particular less than a minute. If the token expires, the curl command will fail with an unauthorized message. So it is important to be prepared.
+--
+
+== Via CLI
+
+include::partial$multi-location/get-bearer-token.adoc[tags=explanation,leveloffset=+1]
+
+:curl_url_1: /graph/v1.0/users/\{username}
+:what_is_listed: Replace the `\{username}` placeholder with the name of the user.
+include::partial$multi-location/get-bearer-token.adoc[tags=common_prep,leveloffset=+1]
+
+=== Get User IDs
+
+:curl_url_1: /graph/v1.0/users/\{username}
+:curl_url_2: /remote.php/dav/spaces/59ee3b90-3231-4621-81aa-4531d33e7671%24fb9e2625-cdb0-4f21-8a34-db775a976707
+include::partial$multi-location/get-bearer-token.adoc[tags=command_prep,leveloffset=+2]
+
+include::partial$multi-location/get-bearer-token.adoc[tags=command_exec,leveloffset=+2]
+
+==== Output
+
+[source,json]
+----
+{
+  "accountEnabled": true,
+  "displayName": "Albert Einstein",
+  "givenName": "Albert",
+  "id": "4c510ada-c86b-4815-8820-42cdf82c3d51",
+  "onPremisesSamAccountName": "einstein",
+  "surname": "Einstein"
+}
+----
+
+=== Output Interpretation and Usage
+
+* The ID identifying the user is under `givenName` named `id` like:
++
+--
+[source,json]
+----
+"id": "4c510ada-c86b-4815-8820-42cdf82c3d51",
+----
+
+* Copy the ID
+** _excluding_ the surrounding double quotes and
+** _embed_ it in single quotes for any tasks that require a space ID as parameter.
+
+Example:
+
+`"4c510ada-...-42cdf82c3d51"` -> `'4c510ada-...-42cdf82c3d51'`
+
+Single quotes are necessary because the ID can contain a `$` sign which is a special character the shell.
```

Comment:
```suggestion
Single quotes are necessary because the ID can contain a `$` sign which is a special character for the shell.
```

If you make this little change, then it needs to be forward-ported.